docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,192 @@
+[clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+the fork to your local machine.
+
+### Creating a Workflow
+
+`ITKRemoteModuleBuildTestPackageAction` is a central repository for Github Actions
+reusable workflows for ITK external modules. Before creating a workflow, ask yourself
+the following questions:
+1. Does my workflow address a problem for ITK external modules that is not
+   already solved?
+2. Does my workflow provide functionality that will generalize to various ITK external modules?
+3. Does my workflow provide functionality related to continuous integration such as
+   building, packaging, and/or testing steps?
+
+If the answer to all of the above is "yes", then continue on with your new workflow!
+
+Reference
+[Github Actions Reusable Workflow documentation](https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow)
+to get started. You will need to:
+1. Create a workflow in the `.github/workflows` folder;
+2. Define workflow inputs;
+3. Define how jobs will be launched;
+4. Define steps that a given job will take.
+
+### Modifying a Workflow
+
+Workflows will need to be maintained and updated over time as tools
+and technology progress. The best approach to modifying a workflow
+is to try to recreate steps on your local system (if possible),
+identify failures and fixes, then translate fixes to the resuable
+workflow in question. If you encounter an issue that requires
+community input you can ask for help on the
+[ITK community Discourse forum](https://discourse.itk.org/).
+
+## Testing Your Changes
+
+Reusable workflows are intended to _manage continuous integration_ for
+ITK external modules in a reusable manner. At the time of writing
+there are unfortunately no good options for establishing continuous
+integration around proposed changes to reusable workflows themselves,
+therefore it is necessary to place an extra burden on the contributors and
+maintainers of `ITKRemoteModuleBuildTestPackageAction` to schedule tests themselves.
+Fortunately, this translates to severely reduced overhead required of the
+general ITK community in maintaining continuous integration on individual
+external modules.
+
+Changes in reusable workflows should be tested through application to
+ITK external modules in a user fork. The ideal repositories to use
+for testing will depend on the workflow changes under test.
+
+The [ITKSplitComponents repository](https://github.com/InsightSoftwareConsortium/ITKSplitComponents)
+is a good target for testing relatively
+simple changes. ITKSplitComponents is a small header-only external module
+with a minimum testing suite, Python wrappings, and an example Jupyter Notebook.
+Do the following to test against ITKSplitComponents or another ITK external module:
+1. Commit your changes to `ITKRemoteModuleBuildTestPackageAction` and
+   push to make them available on your GitHub user account:
+```sh
+> git commit
```

Comment:
Not sure if ">" is preferred over "$".

Reply:
I have replaced with "$" for consistency with ITK's `CONTRIBUTING.md`.